spec/controllers/admin/organizations_controller_spec.rb

```diff
@@ -4,6 +4,7 @@
   let!(:admin) { create(:admin) }
   let!(:organization) { create(:organization) }
   let!(:user) { create(:user) }
+  let!(:user1) { create(:user, email: 'user1@osem.io') }
```

Comment:
Oh! `user` was used in the context `logged in as user with no role` so I thought it better not to assign org admin role to user and create a new object `user1` instead. Although it won't make a difference because we are using `let!`. 
Your question made me realize that I should avoid that!